supervisord.conf

```diff
@@ -0,0 +1,26 @@
+[supervisord]
+nodaemon=true
+
+[program:xl2tpd]
+command=/usr/sbin/xl2tpd -c /etc/xl2tpd/xl2tpd.conf -D
+redirect_stderr=true
+numprocs=1
+stdout_logfile=/dev/fd/1
+stdout_logfile_maxbytes=0
+
+[program:ipsec]
```

Comment:
Thinking of the container as a "single service", I think it makes sense to exit the container when one of the required processes failed. This means that something in the "service" went wrong, and it should stop working.

How to handle that failure is the responsibility of the user; whether that's through `--restart always`, which can be recommended; or through some other orchestration tool (e.g. kubernetes, docker health checks, docker-compose, etc.)